txscript/standard.go

```diff
@@ -512,6 +528,18 @@ func ExtractPkScriptAddrs(pkScript []byte, chainParams *chaincfg.Params) (Script
 			addrs = append(addrs, addr)
 		}
 
+	case ScriptHash32Ty:
+		// A pay-to-script-hash32 script is of the form:
+		//  OP_HASH256 <scripthash> OP_EQUAL
+		// Therefore the script hash is the 2nd item on the stack.
+		// Skip the script hash if it's invalid for some reason.
+		requiredSigs = 1
+		addr, err := bchutil.NewAddressScriptHash32FromHash(pops[1].data,
```

Comment:
Debug log here as well on error?

Reply:
Also do we need a length check or is [1] always going to exist?

Reply:
I just went with the code above it. But added log in ddd1661eefa835f5d028dd4c595b2d7e3977e88e . We might want to add log for the older code above it too. Not sure if any of it is needed really though.

Yeah [1] always exists.